drivers/pds/pdsbackup/backuputils.go

```diff
@@ -237,16 +306,16 @@ func InitializePdsBackup() (*BackupClient, error) {
 	apiConf.Scheme = endpointURL.Scheme
 
 	apiClient := pds.NewAPIClient(apiConf)
-	components := pdsapi.NewComponents(apiClient)
```

Comment:
Add a TODO to use a Init method in pds.go

Reply:
Done